.github/workflows/aks_deploy_review.yml

```diff
@@ -0,0 +1,58 @@
+name: "Deploy review"
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Deploy environment"
+        required: true
+        default: review_aks
+        type: environment
+        options:
+          - review_aks
+  push:
+    branches:
+      - main
+
+  pull_request:
+    branches:
+      - main
+    types:
+      - labeled
+      - synchronize
+      - reopened
+      - opened
+
+jobs:
+  docker:
+    name: Build and push Docker image
+    runs-on: ubuntu-latest
+
+    outputs:
+      image: ${{ steps.build-docker-image.outputs.image }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: DFE-Digital/github-actions/build-docker-image@master
+        id: build-docker-image
+        with:
+          docker-repository: ghcr.io/dfe-digital/early-careers-framework
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  deploy_review:
```

Comment:
could we add smoke tests to the steps? And maybe the step which adds a comment to the pr?

Reply:
Was going to do those as a follow up as we'll be deploying in parallel for a while anyway.